rundeckapp/grails-app/controllers/rundeck/controllers/PluginController.groovy

```diff
@@ -138,10 +138,15 @@ class PluginController extends ControllerBase {
     }
 
     def listPlugins() {
+        String service = params.service
+
         def providers = []
         pluginApiService.listPlugins().each { svc ->
+            if (service && service != svc.service)
```

Comment:
perhaps this needs an API version bump and version check here

Reply:
This should function the same without the new, optional service parameter supplied. Are we still bumping API version for non-breaking changes?

Reply:
the api bump is so that folks using previous versions don't see the API docs and expect the results returned here. so in this case it's primarily to communicate a required version level for the new features, allowing them to be documented accurately